### PR TITLE
Add i18n, with optional language switching

### PR DESCRIPTION
**Pull request by lpancescu**
Added i18n support, with small svg flags for switching (only the USA and Germany flags have been added to the theme - users will have to add their own flags).

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -59,6 +59,11 @@ <h3 class="self-intro">{{ .Site.Params.selfintro }}</h3>
         {{ if .Site.RSSLink }}
             <a href="{{ .Site.RSSLink }}"><img class="icon" src="/img/rss.svg" alt="rss" /></a>
         {{ end }}
+	{{ if and .Site.Params.i18n_flags .IsTranslated }}
```
Comment (nishanths) [inline layouts/partials/header.html:62]: Let's add `i18n_flags` to `config.toml`.

File: static/img/flags/de.svg [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (nishanths) [inline static/img/flags/de.svg:1]: We should plan to optimize these svgs using `svgo` https://github.com/ajstarks/svgo, but we can always do that later :)

**Review by nishanths — COMMENTED** [review]
One comment: Add `i18n_flags` to the example site's `config.toml`. 

Besides that, merge whenever you're ready!

File: static/img/flags/de.svg (hunk 2 above)
Comment (lpancescu) [reply static/img/flags/de.svg:1]: That seems to be a Go library for programatically creating SVGs. Is it possible that you meant https://github.com/svg/svgo (the Node.js optimizer for SVGs)?

**Comment by lpancescu** [comment]
I added `i18n_flags=false` to `examplesite/config.toml`.

**Pull request merged**